src/test/spec/sessions.rs

```diff
@@ -6,20 +6,13 @@ use crate::{
     test::{run_spec_test, TestClient, LOCK},
 };
 
-use super::{run_unified_format_test, run_v2_test};
+use super::run_unified_format_test;
 
 #[cfg_attr(feature = "tokio-runtime", tokio::test(flavor = "multi_thread"))]
 #[cfg_attr(feature = "async-std-runtime", async_std::test)]
 async fn run_unified() {
     let _guard: RwLockWriteGuard<()> = LOCK.run_exclusively().await;
-    run_spec_test(&["sessions", "unified"], run_unified_format_test).await;
-}
-
-#[cfg_attr(feature = "tokio-runtime", tokio::test(flavor = "multi_thread"))]
-#[cfg_attr(feature = "async-std-runtime", async_std::test)]
-async fn run_legacy() {
-    let _guard: RwLockWriteGuard<()> = LOCK.run_exclusively().await;
-    run_spec_test(&["sessions", "legacy"], run_v2_test).await;
```

Comment:
retryable reads/transactions are still using the V2 runner so we can't delete any runner code yet